pkg/virt-operator/resource/generate/rbac/operator.go

```diff
@@ -298,27 +298,6 @@ func NewOperatorClusterRole() *rbacv1.ClusterRole {
 					"get", "list", "watch", "create", "delete", "update", "patch",
 				},
 			},
-			// Until v0.43 a `get` verb was granted to these resources, but there is no get endpoint.
-			// The get permission needs to be kept on the operator level so that updates work.
-			{
-				APIGroups: []string{
-					"subresources.kubevirt.io",
-				},
-				Resources: []string{
-					"virtualmachineinstances/pause",
-					"virtualmachineinstances/unpause",
-					"virtualmachineinstances/addvolume",
-					"virtualmachineinstances/removevolume",
-					"virtualmachineinstances/freeze",
-					"virtualmachineinstances/unfreeze",
-					"virtualmachineinstances/softreboot",
-					"virtualmachineinstances/portforward",
-				},
-				Verbs: []string{
-					"update",
```

Comment:
I think we still need `update`. It's just `get` that can be removed.

Reply:
Yeah that's working, I've amended the commit. Let me know if you still think this is wrong.

Reply:
@lyarwood my impression always was that we have to give virt-operator access to all resources which it would ever want to create RBAC roles and such for, or it may be blocked by security mechanisms. E.g. built-in, PSP, SCC, ...

It may however be the case that this check only extends to service accounts which it wants to use and that we are good.

Reply:
So for instance I have seen typically denies when we would bind a service account to virt-controller which can use /freeze for snapshot orchestration, in case that virt-operator does not have the permissions itself to use /freeze. I may mix up something though.

Reply:
You're correct but we handle that in code with the following call while generating the operator ClusterRole:

https://github.com/kubevirt/kubevirt/blob/c4b6ae63c5a7f642ab86b0755dabca3b814ecb39/pkg/virt-operator/resource/generate/rbac/operator.go#L339-L340

Reply:
https://github.com/kubevirt/kubevirt/blob/43aa19c2b5417c9843f6f7ed58125391cd79c6af/manifests/generated/operator-csv.yaml.in#L591-L600

Reply:
Right. Perfect. I think we are good then :)

Reply:
ha, yeah i get it now. That was tough to wrap my head around at first